### ISSUE TITLE
Comparision of pdfplumber with related libraries

### ISSUE THREAD

**Issue by MartinThoma** [jsvine/pdfplumber#257]
The PDF tooling world is huge. For a beginner it is hard to tell which library to use. It would be nice to have a comparison.

* pdfplumber is built on top of pdfminer.six.  What are examples of core features pdfplumber adds?
* There is also [pymupdf](https://github.com/pymupdf/PyMuPDF) which seems to have a similar feature set and it's just way faster.
* There are a couple of other Python libraries which seem not to be maintained: PyPDF2, PyPDF3, PyPDF4, lapdf
* And more: [pikepdf](https://pypi.org/project/pikepdf/), pdfrw
* And some specialized on tables: Camelot, pdftables, pdftables.six

Is it possible to combine tools, e.g. use logic from camelot to recognize tables and create a HTML output which has semantic HTML tables instead of absolutely positioned cells? (related [question on Stackoverflow](https://stackoverflow.com/q/63508435/562769))

**Comment by MartinThoma** [comment]
Thank you so much! I think once this PR is merged, this issue can be closed :-)

**Comment by jsvine** [comment]
Thanks @MartinThoma, I thought that was a great idea. I've added a brief comparison, as you can see, and will close this issue. But if there are any lingering questions about the differences, don't hesitate to add a comment here.
